.husky/pre-push

```diff
@@ -4,5 +4,4 @@
 yarn check
 yarn test
 
-yarn pylint:check
```

Comment:
Noticed that [someone added this](https://github.com/refstudio/refstudio/blob/main/package.json#L11) to `yarn check` (thank you!), so we don't need to call it twice here